modules/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/actions/data/sql/client/AbstractSQLAction.java

```diff
@@ -365,14 +365,21 @@ private CallableStatement getPreparedCall(Connection conn, SQLDatasource datasou
     private ArrayList<ColumnDefinition> getColumnDefinitions(ResultSet rs)
```

Comment:
The abstract type should be mentioned here, i.e. List. The same goes for the local variables in the method (LHS references), they should be fixed as well.

Reply:
Fixed with a56e89440acb2461848b604b4d25ef85f0862843. Thanks